build.proj

```diff
@@ -50,9 +51,10 @@
       <InputOSGroup>$(InputOSGroup)</InputOSGroup>
     </Project>
     <Project Include="src\post.builds">
```

Comment:
See https://github.com/dotnet/corefx/pull/14098 it covers part of this. I expect that post and sign will still be later steps in the build.proj but packages will be done via build-packages.cmd